src/transcoding/codec/profile_audio_class.c

```diff
@@ -358,7 +360,7 @@ const codec_profile_class_t codec_profile_audio_class = {
                 .group    = 4,
                 .opts     = PO_ADVANCED | PO_PHIDDEN,
                 .get_opts = codec_profile_audio_class_channel_layout_get_opts,
-                .off      = offsetof(TVHAudioCodecProfile, channel_layout),
+                .off      = offsetof(TVHAudioCodecProfile, ch_layout_u_mask),
```

Comment:
This will only exist for new ffmpeg versions, compilation will fail on old versions

Reply:
fixed.